K8s-deployment/misc/K8s-ServersRestart-Script/K8s-API-Servers-restart.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+
+i=0
+#reads all namespaces listed in endpoint.json file
+
+cat $1 | jq -r .namespaces[] | while read ns ;
+do 
+        kubectl delete po -n $ns --all --dry-run=client
+        if [ $? == 0 ];then
+                echo -e "\033[31m pods in $ns namespace are deleted\033[0m"
+                echo waiting for pods to restart....
+                sleep 30
+        fi
+        if [ $ns == "lip" ] || [ $ns == "auditing" ];then
+                sleep 60
+                ((i++))
+                continue
+        else
+        #reads all API server endpoints listed in endpoint.json file to check endpoint's health
+
+        cat $1 | jq -r .endpoints[$i] > endpoint.txt
+        while read endpoint; 
```

Comment:
Any particular reason why loop inside loop is used?
Can be done in simpler manner, probably?
1. Read each namespace in sequential manner , i=0 to n 
and delete pods
2. For that namespace (that index i) check the endpoint 
if it exists

Reply:
Suggested changes have been made @abhi4578 .